app/routes/oppgave.$oppgaveId.oversikt.tsx

```diff
@@ -0,0 +1,8 @@
+import OpplysningForslag from "~/components/opplysning-forslag/OpplysningForslag";
+import { useTypedRouteLoaderData } from "~/hooks/useTypedRouteLoaderData";
+
+export default function Oppgave() {
+  const { behandling } = useTypedRouteLoaderData("routes/oppgave.$oppgaveId");
+
+  return <OpplysningForslag opplysninger={behandling.opplysning} />;
```

Comment:
Routen må wrappes in Tabs.Panel for at tabs skal funke riktig
```suggestion
    return  (
        <Tabs.Panel value="oversikt">
      <OpplysningForslag opplysninger={behandling.opplysning} />
    </Tabs.Panel>
    );
```